spec/views/users/users_show_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :system do
```

Comment:
- You have done well so far, you have included all the required tests according to the project requirements, but all your tests are failing, this is because you are using the wrong test type while testing. Since this is an integration test that require you to test the behavior of your app, please use `type: :feature` in place of `type: :system`. Please fix this 👍 👍 

![wfqwrfqw](https://github.com/eamador1/blog-app/assets/73607512/fdaef4df-e2f0-40db-b854-430101218a69)